common/persistence/nosql/nosqlplugin/cassandra/workflowUtils.go

```diff
@@ -1171,27 +1200,27 @@ func (db *cdb) updateWorkflowExecutionAndEventBufferWithMergeAndDeleteMaps(
 		return fmt.Errorf("should only support WorkflowExecutionMapsWriteModeUpdate")
 	}
 
```

Comment:
buffered event ttl is missing

Reply:
ahh! do we need ttl in appendBufferedEvents too?